Army/ArmyDarkWarLegion.cs

```diff
@@ -0,0 +1,69 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Army/CoreArmyLite.cs
+//cs_include Scripts/Story/Legion/DarkWarLegionandNation.cs
+//cs_include Scripts/Story/LordsofChaos/Core13LoC.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Models.Quests;
+
+public class ArmyDarkWarLegion
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv => new();
+    private CoreArmyLite Army = new();
+    public DarkWarLegionandNation DW = new();
+
+    private static CoreBots sCore = new();
+    private static CoreArmyLite sArmy = new();
+
+    public string OptionsStorage = "ArmyDarkWarLegion";
+    public bool DontPreconfigure = true;
+    public List<IOption> Options = new List<IOption>()
+    {
+        sArmy.player1,
+        sArmy.player2,
+        sArmy.player3,
+        sArmy.player4,
+        sArmy.player5,
+        sArmy.player6,
+        sArmy.player7,
+        sArmy.packetDelay,
+        sCore.SkipOptions
+    };
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions(disableClassSwap: true);
+        bot.Options.RestPackets = false;
+
+        if (!Bot.Quests.IsUnlocked(8584))
```

Comment:
use Core.IsCompletedBefore(); for checking storyline completeness

Reply:
Added!